doc/sphinx/proof-engine/ltac.rst

```diff
@@ -31,7 +31,7 @@ Syntax
 The syntax of the tactic language is given below. See Chapter
 :ref:`gallinaspecificationlanguage` for a description of the BNF metasyntax used
 in these grammar rules. Various already defined entries will be used in this
-chapter: entries :token:`natural`, :token:`integer`, :token:`ident`,
+chapter: entries :token:`natural`, :token:`int`, :token:`ident`,
```

Comment:
Token `natural` is defined in the SSReflect chapter as a `num` or an `ident`. It is used very seldom (not in the SSReflect chapter which uses `num` instead, only here and in the example given in the Sphinx README). We should probably remove it altogether, although I would be interested to understand in which case it is allowed to use a variable instead of a `num` and to document it.

Reply:
`natural` is not defined as a nonterminal in the ssr grammar.  It's defined in g_prim.mlg and only allows digits, no ident.